### PR TITLE
Fix LICENSE on GitHub

### PR DESCRIPTION
**Pull request by dschuermann**
GitHub displays "BSD-3-Clause" as K-9 Mail's license in the header above the file listing. This is because the repo contains the file "HTMLCLEANER_LICENSE". Adding a "LICENSE" file with Apache v2 should solve this issue.

### CODE REVIEW

**Comment by philipwhiuk** [comment]
Do we need NOTICE and LICENSE?

**Comment by dschuermann** [comment]
@philipwhiuk Like this?

File: NOTICE [hunk 1]
```diff
@@ -1,190 +1,41 @@
...
-      of any other Contributor, and only if You agree to indemnify,
-      defend, and hold each Contributor harmless for any liability
-      incurred by, or claims asserted against, such Contributor by reason
-      of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
-
+K-9 Mail
+Copyright 2008-2016, K-9 Mail Developers
+Copyright 2006-2016, The Android Open Source Project
```
Comment (philipwhiuk) [inline NOTICE]: I think this should be 2005 per line 2 of the old NOTICE.

**Comment by philipwhiuk** [comment]
Looks a lot better. Thanks for looking into this @dschuermann +1 Probably need @cketti to review this as project lead but it's almost there I think.

**Comment by cketti** [comment]
I don't think we need to include the HtmlCleaner license in the NOTICE file. The repository doesn't contain any HtmlCleaner code. And we do have attribution for the Open Source libraries the app depends on in the About dialog of the app.

File: NOTICE (hunk 1 above)
Comment (dschuermann) [inline NOTICE]: I took the years from the About screen, but you are right old NOTICE says 2005. Will change it.

**Comment by dschuermann** [comment]
done

**Comment by cketti** [comment]
Thanks

**Pull request merged**